lib/y2lan_restart_common.pm

```diff
@@ -71,8 +71,8 @@ sub initialize_y2lan
     assert_script_run 'sed -i -E \'s/' . $debug_conf . '/\' /etc/sysconfig/network/config';
     assert_script_run 'systemctl restart network';
     # Keeping "shutting down|ifdown all" for compatibility with NetworkManager and previous wicked versions.
-    type_string "journalctl -f|egrep -i --line-buffered 'shutting down|ifdown all|Stopping wicked' > journal.log &\n";
-    assert_script_run '> journal.log';     # clear journal.log
+    type_string "journalctl -f|egrep -i --line-buffered 'shutting down|ifdown all|Stopping wicked|Reloaded wicked' > journal.log &\n";
+    clear_journal_log();
```

Comment:
where is declared?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/y2lan_restart_common.pm#L397

Reply:
sorry, my search in the editor got stuck somehow!